examples/simulator-nft-dumb/index.rsh

```diff
@@ -0,0 +1,53 @@
+'reach 0.1';
```

Comment:
Don't you have a Makefile that normally copies these?

Reply:
yes, but i don't use it locally, so sometimes i forget before pushing, sorry, uno momento

Reply:
i just forgot to push this time